sys-libs/pam/files/tmpfiles.d/pam.conf

```diff
@@ -8,3 +8,4 @@ L /etc/security/limits.conf	-    -	  -	-	../../usr/lib/pam/limits.conf
 L /etc/security/namespace.conf	-    -	  -	-	../../usr/lib/pam/namespace.conf
 L /etc/security/pam_env.conf	-    -	  -	-	../../usr/lib/pam/pam_env.conf
 L /etc/security/time.conf	-    -	  -	-	../../usr/lib/pam/time.conf
+C /etc/pam.d			-    -	  -	-	/usr/lib/pam.d
```

Comment:
Why is this a copy? Doesn't pam read lib/pam.d directly now? In theory `d /etc/pam.d` is the only rule needed.